src/components/learner-credit-management/tests/CatalogSearch.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import {
+  SEARCH_FACET_FILTERS,
+  SearchContext,
+} from '@edx/frontend-enterprise-catalog-search';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { screen } from '@testing-library/react';
+import { renderWithRouter } from '../../test/testUtils';
+import CatalogSearch from '../search/CatalogSearch';
+
+jest.mock('react-instantsearch-dom', () => ({
+  ...jest.requireActual('react-instantsearch-dom'),
+  InstantSearch: () => <div>SEARCH</div>,
+  Index: () => <div>SEARCH</div>,
+}));
+
+const DEFAULT_SEARCH_CONTEXT_VALUE = { refinements: {} };
+
+// eslint-disable-next-line react/prop-types
```

Comment:
On `master`, I've updated this repo's eslint config to ignore all prop-type ESLint warnings like this across all test files. The ESLint disable should no longer be necessary once rebased! 😄